run

```diff
@@ -59,18 +59,24 @@ while read f; do
 	echo -n "checking $f ... "
 	check "$name" "$f" > output/$name.out || true
 	sed -i "s/^$name\.[^:]\+: /$name: /" output/$name.out
-	if diff -u tests/$name.ref output/$name.out; then
-		echo "ok"
-		: $((++good))
-	else
-		echo "fail"
-		if [ ! -e tests/$name.ref ]; then
+	reffile="tests/$name.$(uname -i).ref"
```

Comment:
You might rather want `uname -m` here, `uname -i` is not portable I think. On my own host I get `GenuineIntel` as output.

Reply:
thank you for the hint